boxcli/config.go

```diff
@@ -0,0 +1,18 @@
+package boxcli
+
+import (
+	"github.com/spf13/cobra"
+)
+
+const currentDir = "."
+
+// to be composed into xyzCmdFlags structs
+type configFlags struct {
+	path string
+}
+
+func registerConfigFlags(cmd *cobra.Command, flags *configFlags) {
+	cmd.Flags().StringVarP(
+		&flags.path, "config", "c", currentDir, "path to directory containing a devbox.json config file",
```

Comment:
If you need to distinguish between the flag being set and not set, you might want the default value to actually be the empty string.

Reply:
cobra has an api something like `if cmd.Flags().Changed(flagName) {`

I suppose it depends on where I need to distinguish this.